copulas/univariate/base.py

```diff
@@ -258,68 +281,22 @@ def fit(self, X, *args, **kwargs):
             else:
                 self.model = getattr(scipy.stats, self.model_class)(X, *args, **kwargs)
 
-            for name, method_name in self.method_map.items():
-                if method_name is None:
-                    method = getattr(self, name)
-                    setattr(self, name, missing_method_scipy_wrapper(method))
+            method_names = [
```

Comment:
I think this could be a class level attribute, `self.METHOD_NAMES`. I also should be a tuple (immutable)

Reply:
Right, already changed.